src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.Configuration.Tests/ConfigurationLiveTests.cs

```diff
@@ -34,8 +34,14 @@ public class ConfigurationLiveTests
 
         private static void AssertEqual(ConfigurationSetting expected, ConfigurationSetting actual)
```

Comment:
I would suggest that you use the KeyValue's equals method instead of re-implementing equals in the test.

Reply:
Makes sense. Done